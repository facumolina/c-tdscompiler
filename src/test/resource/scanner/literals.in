/* Int literals */
1
01
12398891

/* Non Int literals */
nasdasd1
/*111*/

/* Float literals */
0.121
111.2133

/* Non Float literals*/
.12313
123123...

/* Boolean literals */
true
false

/* Non Boolean literals*/
atrue
false123