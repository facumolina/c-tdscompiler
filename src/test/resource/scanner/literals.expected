COMMENT
INT_LITERAL
INT_LITERAL
INT_LITERAL
COMMENT
ID
COMMENT
COMMENT
FLOAT_LITERAL
FLOAT_LITERAL
COMMENT
DOT
INT_LITERAL
INT_LITERAL
DOT
DOT
DOT
COMMENT
TRUE
FALSE
COMMENT
ID
ID